OPAL/tac/src/main/scala/org/opalj/tac/fpcf/analyses/cg/AndroidICCAnalysis.scala

```diff
@@ -319,12 +314,22 @@ class AndroidICCAnalysis(val project: SomeProject) extends FPCFAnalysis {
         val callbackList = source.getLines.toList
         source.close
         val dec = project.get(DeclaredMethodsKey)
+        val leftOverMethods = mutable.Queue[Method]()
         project.allMethodsWithBody.filter(dec(_) != null).foreach(m ⇒ {
             val tacaiEP = propertyStore(m, TACAI.key)
             if (tacaiEP.hasUBP && tacaiEP.ub.tac.isDefined) {
                 processMethod(m, callbackList)
-            }
+            } else leftOverMethods += m
         })
+        while (leftOverMethods.size > 1) {
```

Comment:
This busy waiting is an extremely bad idea. You absolutely have to handle this using a continuation function.